libs/platform/experience/src/services/layout/plugins/styles/misc/misc-layout.model.ts

```diff
@@ -0,0 +1,110 @@
+     * layout, by taken the parent item _span_ count, which default to `1`. This results in a
+     * balanced layout, where nested layouts will reflect the parent column setup.
+     */
+    columnCount?: number;
+
+    /**
+     * The align property is applied to the child items as well as to the layout it self. This
+     * allows to align the child items over the layout axis. In most layouts, this means that
+     * child items are aligned over the vertical axis. For example, the item(s) are stretched
+     * or centered.
+     */
+    align?: LayoutAlign | string;
+    justify?: LayoutAlign | string;
+
+    /**
+     * Makes it possible for an element to span across the given columns or rows.
+     *
+     * The span size might exceed the available columns, due to the dynamic/responsive
+     * behaviour of the layout. Whenever the span is applied, it is therefore combined
+     * with the maximum available columns to span.
+     */
+    span?: number;
+    colSpan?: number;
+    rowSpan?: number;
+
+    /**
+     * Allows to specify the location of the grid row. This overrules the automated
+     * flow of the grid system.
+     *
+     * A separate grid item that is used in combination with a list of items can interfere with
+     * this property with the list, i.e. by setting the `gridRow` to 2, it will be nested inside
+     * the list on the 2nd row.
+     */
+    gridRow?: number;
+
+    /**
+     * Allows to specify the location of the grid column. This overrules the automated
+     * flow of the grid system.
+     *
+     * A separate grid item that is used in combination with a list of items can interfere with
+     * this property with the list, i.e. by setting the `gridColumn` to 3, it will be placed on
+     * the 3rd column.
+     */
+    gridColumn?: number;
+
+    /**
+     * Sets the gaps (gutters) between rows and columns. The gap is applied to all layouts.
+     */
+    gap?: string | number;
+
+    /**
+     * Specifies the stack order of a component. This is useful in combination with sticky layouts,
+     * where the stacking order cannot be controlled by static css rules.
+     *
+     * Defaults to `1` in sticky layouts.
+     */
+    zIndex?: number;
+
+    /**
+     * Very useful for stick layouts that need to allow for scrollable overflow, typically
+     * in combination with sticky layout. The side navigation aside the page content is a
+     * good example where this is needed, since the aside navigation is often setup to stick
+     * to the top so that it won't scroll away.
+     *
+     * The overflow cannot be hardcoded for sticky layouts, since there are sticky layouts that
+     * require explicit overflows (eg. typeahead search inside a sticky header)
+     */
+    overflow?: string;
+
+    /**
+     * Specifies the background of the item. The background can be used to set a color or even
+     * a background image.
+     */
+    background?: string;
```

Comment:
let's move background, fill, border and radius to `canvas` styles plugin. We might come with a better name over time. We're on our way to eliminate `misc`.